fk/teamcreate.go

```diff
@@ -139,8 +154,25 @@ func teamCreate() exitCode {
 
 	printHeader("Finishing setup")
 
-	out.Print("You've indicated you want setup " + teamName + " using your key\n")
-	out.Print(fmt.Sprintf("%s\n", key.Fingerprint()))
+	uuid, err := uuid.NewV4()
+	if err != nil {
+		out.Print(ui.FormatFailure("Error creating UUID for team", nil, err))
+		return 1
+	}
+
+	t := team.Team{
+		UUID:   uuid,
+		Name:   teamName,
+		People: teamMembers,
+	}
+
+	printCheckboxPending("Create team roster")
+	err = team.Save(t, fluidkeysDirectory)
+	if err != nil {
+		printCheckboxFailure("Create team roster", err)
+	}
+	printCheckboxSuccess("Create team roster in \n" +
```

Comment:
The alignment of the second line here is a bit borked 
![image](https://user-images.githubusercontent.com/254290/53340882-00581080-3902-11e9-9d03-8d44bd5dc175.png)

Reply:
Ah yes! Will fix!